attitude_error_mon/att_err_mon.py

```diff
@@ -0,0 +1,354 @@
+            density=True,
+            color="b",
+            alpha=0.4,
+            label=f"{d0_str} to {d1_str}",
+        )
+        plt.hist(
+            recent_data[f"{ax}_err"],
+            bins=bins,
+            log=True,
+            density=True,
+            color="r",
+            alpha=0.4,
+            label=f"{d2_str} to {d3_str}",
+        )
+        plt.xlabel(f"{ax} err (arcsec)")
+        plt.legend(loc="upper right", fontsize=7)
+        plt.title(f"per obs 99th percentile {ax} err\n({msid})", fontsize=12)
+        plt.grid()
+        plt.tight_layout()
+        plt.savefig(os.path.join(outdir, f"{ax}_err_hist.png"))
+
+
+def update_file_data(data_file, start, stop):
+    if os.path.exists(data_file):
+        print(f"Reading previous data from {data_file}")
+        last_data = Table.read(data_file, format="ascii")
+        new_data = get_obs_table(last_data[-5]["date"], stop)
+        if new_data["date"][0] > last_data["date"][-1]:
+            data = vstack([last_data, new_data])
+        else:
+            idx_old_data = np.flatnonzero(last_data["date"] >= new_data["date"][0])[0]
+            data = vstack([last_data[0:idx_old_data], new_data])
+        data = data[data["date"] >= start.date]
+    else:
+        data = get_obs_table(start, stop)
+    data.sort("date")
+    data.write(data_file, format="ascii", overwrite=True)
+    return data
+
+
+def update(datadir, outdir, full_start, recent_start, point_lim=7.5, roll_lim=15):
+    data_file = os.path.join(datadir, "data.dat")
+    dat = update_file_data(data_file, full_start, DateTime())
+    recent_data = dat[dat["time"] >= recent_start.secs]
+    ref_data = dat[dat["time"] < recent_start.secs]
+
+    one_shot_plot(ref_data, recent_data, outdir=outdir)
+
+    comments_file = os.path.join(outdir, "comments.dat")
+    if os.path.exists(comments_file):
+        comments = ascii.read(comments_file, Reader=ascii.FixedWidthTwoLine)
+        # convert to a dict
+        comments = {comment["obsid"]: comment["comment"] for comment in comments}
+    else:
+        comments = {}
+
+    ok = (recent_data["point_err"] < point_lim) | (recent_data["roll_err"] < roll_lim)
+    outliers = recent_data[~ok]
+    comments_col = Column(
+        data=[comments.get(row["obsid"], "") for row in outliers], name="comment"
+    )
+    outliers.add_column(comments_col)
+    recent_data = recent_data[ok]
+
+    ok = (ref_data["point_err"] < point_lim) | (ref_data["roll_err"] < roll_lim)
+    ref_data = ref_data[ok]
+    att_err_time_plots(ref_data, recent_data, outdir=outdir)
+    att_err_hist(ref_data, recent_data, outdir=outdir)
+
+    template_html = open(os.path.join(FILE_DIR, "data", "index_template.html")).read()
+    template = jinja2.Template(template_html)
+    out_html = template.render(outliers=outliers, one_shot_start=ref_data["date"][0])
+    with open(os.path.join(outdir, "index.html"), "w") as fh:
+        fh.write(out_html)
+
+
+def main(args=None):
+    opt = get_options()
```

Comment:
This is where I would parse the arguments when returning the parser in `get_options`
```suggestion
    opt = get_options().parse_args()
```